contextual_lenses/load_transformer.py

```diff
@@ -0,0 +1,31 @@
+"""Utils for loading FlaxLM or FlaxBERT models or parameters."""
+
+
+import os
+import tensorflow as tf
+import gin
+from protein_lm import data, models
+
+
+def load_transformer_model(ckpt_dir, model_cls, domain=None):
```

Comment:
this function should definitely have test coverage. The easiest way to do this would be to make a new small model, compute the output on some dummy inputs, save the model to disk, make a new model by loading from file, and check that the output on the dummy inputs is the same.